From: Aaron Puchert <aaronpuchert@alice-dsl.net>
Date: Sun, 9 Apr 2023 01:38:17 +0200
Subject: [PATCH 13/20] Resolve ambiguous call to tmOnAssert

The conversion to `const wxChar*` was probably implicit previously, if
needed at all.
---
 Source/tmHeader.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Source/tmHeader.cpp b/Source/tmHeader.cpp
index 53eda36..edd6b52 100644
--- a/Source/tmHeader.cpp
+++ b/Source/tmHeader.cpp
@@ -113,7 +113,7 @@ void tmOnAssert(int cond, const char* szFile, int nLine,
   wxString strFile;  strFile.FromAscii(szFile);
   wxString strCond;  strCond.FromAscii(szCond);
   wxString strMsg;   strMsg.FromAscii(szMsg);
-  tmOnAssert(cond, strFile.c_str(), nLine, strCond.c_str(), strMsg.c_str());
+  tmOnAssert(cond, strFile.c_str().AsWChar(), nLine, strCond.c_str().AsWChar(), strMsg.c_str().AsWChar());
 }
 #endif // wxUSE_UNICODE
 
